mix.exs

```diff
@@ -50,6 +50,8 @@ defmodule Advisor.Mixfile do
   defp aliases do
     ["ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
      "ecto.reset": ["ecto.drop", "ecto.setup"],
-     "test": ["ecto.create --quiet", "ecto.migrate", "test"]]
+     "test": ["ecto.create --quiet", "ecto.migrate", "test"],
+     "ci": ["test", "credo --ignore readability"]
```

Comment:
excellent catch. Initially I thought if leaving it off, but by now I have fixed all issues concerning readability. I'll fix it tonight.